test/ci/MarkdownPreviewTest.tsx

```diff
@@ -0,0 +1,80 @@
+/**
+ * Test the Markdown-preview plugin
+ */
+
+import * as assert from "assert"
+import * as Oni from "oni-api"
+
+interface IPlugin {
+    /**
+     * Gets the current value of a property.
+     * The return type is unknown.
+     *
+     * param: name  A name of a property
+     * returns      The current value of the specified property
+     */
+    getStatus(name: string): any
+}
+
+interface IPluginManager {
+    getPlugin(name: string): Promise<IPlugin | null>
+}
+
+interface IOniWithPluginApi {
+    plugins: IPluginManager
+}
+
+export async function test(typedOni: Oni.Plugin.Api) {
+    const typelessOni = typedOni as any
+    const oni = typelessOni as IOniWithPluginApi
+
+    const markdownPlugin = await oni.plugins.getPlugin("oni-plugin-markdown-preview")
+    assert.notEqual(markdownPlugin, null)
+    if (markdownPlugin === null) {
+        return
+    }
+
+    describe("preview pane", async () => {
```

Comment:
Interesting - did you need to do anything special to get `describe` available here? I think this depends on `mocha` and I'm not sure if we have it in the context of these tests, which are really running inside oni

Reply:
As you can see in the PR's diff, there is nothing special - I just assumed it will work.
On the other hand, I didn't really tested it since there in no implementation for `getPlugin` & `getStatus` in Oni yet.